batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/dao/StudentDAO.java

```diff
@@ -0,0 +1,134 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.entity.Student;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.util.DBConnection;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by sanjay on 1/14/16.
+ */
+public class StudentDAO {
+  private static final Logger LOGGER = Logger.getLogger(StudentDAO.class.getName());
+
+  private static final String UPDATE = "UPDATE tbl_userinfo SET firstname=?, middlename=?, lastname=?, address=?, grade=? WHERE id=?";
+  private static final String LIST = "SELECT * FROM tbl_userinfo LIMIT ? OFFSET ?";
+  private static final String INSERT = "INSERT INTO tbl_userinfo (firstname,middlename,lastname,address,grade) VALUES (?,?,?,?,?)";
+  private static final String SELECT_BY_ID = "SELECT * FROM tbl_userinfo WHERE id=?";
+  private static final String DELETE = "DELETE FROM tbl_userinfo WHERE id=?";
+  private static final String COUNT_STUDENTS = "SELECT count(*) FROM tbl_userinfo";
+
+
+  public void insert(Student s) throws DataException//get the object
```

Comment:
it is always a best practice to return the same object after save/edit so that you can use the saved if needed.